go.mod

```diff
@@ -30,11 +30,11 @@ require (
 	github.com/ipfs/go-ipld-format v0.4.0
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ipfs/go-merkledag v0.6.0
-	github.com/libp2p/go-libp2p v0.20.2
+	github.com/libp2p/go-libp2p v0.20.3
 	github.com/libp2p/go-libp2p-core v0.17.0
 	github.com/libp2p/go-libp2p-kad-dht v0.16.0
 	github.com/libp2p/go-libp2p-peerstore v0.6.0
-	github.com/libp2p/go-libp2p-pubsub v0.7.0
+	github.com/libp2p/go-libp2p-pubsub v0.7.1-0.20220701163738-60cf38003244
```

Comment:
can we use a concrete released/tagged version?
https://github.com/libp2p/go-libp2p-pubsub/releases/tag/v0.7.1

Reply:
I would like to, but unfortunately no. @vyzo didn't tag a new release for all the commits in the master, and probably had reasons for that, but maybe he will change his mind ;)

Reply:
hey guys, i can tag a new release for v0.8 by week's end.

The only reason i havent rushed to it is the upgrade to depend on go-libp2p (v20), now that it has monorepoized, so i wanted to give it some bake time.

In the future pls ask when you need a release so that we can coordinate!

Reply:
Awesome, thanks for your response!

We can keep this PR open until a new pubsub release then, no rush from our side.